satpy/readers/vii_base_nc.py

```diff
@@ -18,6 +18,13 @@
 
 """EUMETSAT EPS-SG Visible/Infrared Imager (VII) readers base class."""
 
+#     P.Phillips 14/01/22  Removed the following as resulted in no data on one side of the
+#     Greenwich meridian being lost and not needed for test data V2:
```

Comment:
This comment should be provided in the commit message, as the history of changes is available that way. Please remove this comment altogether.

Reply:
ok, I will remove the comment